docs/source/contributing/developing-core.md

```diff
@@ -9,7 +9,11 @@ myst:
 
 # Develop Volto core
 
-This chapter describes how to develop Volto core and its libraries, packages, and apps as open source software contributions.
+This chapter describes how to develop the latest version of Volto core and its libraries, packages, and apps as open source software contributions.
+
+```{seealso}
+For previous versions of Volto core, you should follow the guide {doc}`plone:contributing/core/index`.
```

Comment:
@stevepiercy This is wrong because the plone contribute to core describes only how to contribute to Plone core using buildout
https://6.docs.plone.org/contributing/core/index.html

In previous versions of Volto we didn't teach that users should use the buildout core to develop either.
We had the same setup to start the backend using the make command although with previous naming.
For me it makes more sense for previous versions of Volto if docker installation isn't preferred to reffer to the api folder which gives info on how to create the backend this way https://github.com/plone/volto/tree/17.x.x/api

Reply:
Alternatively, we don't write anything as the flow is always contributing to the main branch followed by backports if it's not a breaking change.

Reply:
You're right. Plus docs have changed significantly. I updated the text in 076d250778b962f6449a1350ce3d1c7b85c083c9. One more review please!